docs/Makefile

```diff
@@ -32,60 +27,41 @@ endif
 
 MARKDOWNLINKCHECK?=markdown-link-check
 MARKDOWNLINKCHECK_IMG?=ghcr.io/tcort/markdown-link-check:stable
-# if there's no local markdown-link-check, fallback to docker
+
 ifeq (, $(shell ${MARKDOWNLINKCHECK} --version 2> /dev/null))
 ifeq (, $(shell docker version 2> /dev/null))
 ifneq (, $(shell $(NETLIFY) "true"))
```

Comment:
is this if block still required without netlify?

Reply:
Yes, because the Docker fallback has nothing to do with Netlify and is a convenience for a user to not have to install anything on their machines.